projects/1.20/assets/doggy-talents-next/doggytalents/lang/zh_cn.json

```
@@ -0,0 +1,824 @@
+    "item.doggytalents.spotted_collar": "斑点项圈",
+    "item.doggytalents.multicoloured_collar": "彩虹项圈",
+    "item.doggytalents.canine_tracker": "狗狗追踪器",
+    "item.doggytalents.canine_tracker.description": "右键开始追踪",
+    "item.doggytalents.creative_canine_tracker": "创造模式狗狗追踪器",
+    "item.doggytalents.creative_canine_tracker.description": "右键开始追踪",
+    "item.doggytalents.whistle": "哨子",
+    "item.doggytalents.whistle.description": "按住潜行键+右键切换其他哨子模式",
+    "item.doggytalents.whistle.0": "哨子指令：起立",
+    "item.doggytalents.whistle.1": "哨子指令：跟随",
+    "item.doggytalents.whistle.2": "哨子指令：坐下",
+    "item.doggytalents.whistle.3": "哨子指令：停止攻击",
+    "item.doggytalents.whistle.4": "牧羊犬哨子",
+    "item.doggytalents.whistle.5": "战术哨子",
+    "item.doggytalents.whistle.6": "咆哮哨子",
+    "item.doggytalents.whistle.7": "指名召唤哨子",
+    "item.doggytalents.whistle.8": "哨子指令：上床睡觉",
+    "item.doggytalents.whistle.9": "哨子指令：到我身后",
+    "item.doggytalents.whistle.10": "群体召唤哨子",
+    "item.doggytalents.whistle.11": "搬运猎犬哨子",
+    "item.doggytalents.whistle.12": "眼神召唤哨子", 
+    "item.doggytalents.whistle.13": "上车哨子", 
+    "item.doggytalents.whistle.14": "哨子指令：嗷呜！",
+    "item.doggytalents.whistle.15": "全体切换模式",
+    "item.doggytalents.whistle.16": "嘘！安静！",
+    "item.doggytalents.whistle.17": "允许跨维度传送",
+    "item.doggytalents.whistle.18": "床犬哨子",
+    "item.doggytalents.whistle.19": "执勤哨子",
+    "item.doggytalents.whistle.20": "载我哨子",
+    "item.doggytalents.whistle.0.help": "命令所有（仅执勤模式下）坐下的狗狗站立。",
+    "item.doggytalents.whistle.1.help": "命令所有（仅执勤模式下）站立的狗狗传送到你身边。",
+    "item.doggytalents.whistle.2.help": "命令所有（仅执勤模式下）站立的狗狗坐下。",
+    "item.doggytalents.whistle.3.help": "命令所有狗狗停止攻击目标。",
+    "item.doggytalents.whistle.4.help": "牧羊犬技能专用哨子，详见技能说明。",
+    "item.doggytalents.whistle.5.help": "使用时射出弹射物，被击中的实体将成为所有战术犬的目标。",
+    "item.doggytalents.whistle.6.help": "烈风咆哮技能专用哨子，详见技能说明。",
+    "item.doggytalents.whistle.7.help": "按名称召唤特定狗狗到你身边。",
+    "item.doggytalents.whistle.8.help": "命令所有站立的狗狗前往并坐在其认领的狗窝上（如果附近有）。",
+    "item.doggytalents.whistle.9.help": "命令所有站立的狗狗到你身后。",
+    "item.doggytalents.whistle.10.help": "按组别召唤特定狗狗到你身边。",
+    "item.doggytalents.whistle.11.help": "搬运猎犬技能专用哨子，详见技能说明。",
+    "item.doggytalents.whistle.12.help": "召唤你视线所指的狗狗。",
+    "item.doggytalents.whistle.13.help": "当骑乘坐骑且有额外空位时，用此哨子让狗狗与你同乘。",
+    "item.doggytalents.whistle.14.help": "命令你指向的狗狗发出嚎叫。",
+    "item.doggytalents.whistle.15.help": "命令所有跟随的站立狗狗切换至指定模式。",
+    "item.doggytalents.whistle.16.help": "命令所有狗狗暂时保持安静。",
+    "item.doggytalents.whistle.17.help": "允许/禁止所有站立狗狗进行跨维度传送。",
+    "item.doggytalents.whistle.18.help": "床犬技能专用哨子，详见技能说明。",
+    "item.doggytalents.whistle.19.help": "切换所有站立狗狗的执勤状态。",
+    "item.doggytalents.whistle.20.help": "命令附近的骑乘犬强制载你上背（空手双击狗狗）",
+
+    "item.doggytalents.treat_bag": "零食袋",
+    "item.doggytalents.energizer_stick": "能量棒",
+    "item.doggytalents.egg_sandwich": "鸡蛋三明治",
+    "item.doggytalents.egg_sandwich.description": "最棒的三明治！雷格西与雷格姆认证！",
```

```suggestion
    "item.doggytalents.egg_sandwich.description": "最棒的三明治！雷格西与蕾格姆认证！",
```
大陆地区翻译为“[蕾格姆](https://baike.baidu.com/item/BEASTARS/20402882)”